pkg/providers/anthropic/chat_stream.go

```diff
@@ -1,16 +1,85 @@
 package anthropic
 
 import (
-	"context"
+    "context"
+    "encoding/json"
+    "fmt"
+    "io"
+    "net/http"
 
-	"glide/pkg/api/schemas"
-	"glide/pkg/providers/clients"
+    "glide/pkg/api/schemas"
+    "glide/pkg/providers/clients"
+    "glide/pkg/telemetry"
+    "go.uber.org/zap"
 )
 
 func (c *Client) SupportChatStream() bool {
-	return false
+	return true
 }
 
-func (c *Client) ChatStream(_ context.Context, _ *schemas.ChatRequest) (clients.ChatStream, error) {
```

Comment:
There has to be `ChatStream()` method that would create an instance of AnthropicChatStream in this case.
For example, in OpenAI case, it looked this way: 
https://github.com/EinStack/glide/blob/develop/pkg/providers/openai/chat_stream.go#L162-L177

Without that there is nothing that would use the AnthropicChatStream struct